ui/src/components/ClubActiveEvents/index.tsx

```diff
@@ -0,0 +1,40 @@
+import styles from './styles.module.scss';
+
+interface IClubActiveEvents {
+  club: string;
+}
```

Comment:
Is this necessary?

Reply:
I have removed this prop and instead of this, we can filter the club's active events with URL params.